How-To/Connect-to-AWS-Resources.md

```diff
@@ -142,19 +143,20 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
     df = cursor.as_pandas()
     ```
 
-### Tableau
+### Tableau and Tableau Prep
 
 You will likely need to work with IT admins for permissions to do the following.
 
-1. Install [Tableau Desktop version 2022.1](https://www.tableau.com/support/releases/desktop/2022.1.23) if you plan to publish a file to the CCAO's Tableau Server. The version of Tableau Desktop [cannot be more recent](https://help.tableau.com/current/desktopdeploy/en-us/desktop_deploy_version_compat_top.htm) than the version of Tableau Server.
+1. Install [Tableau Desktop](https://www.tableau.com/support/releases) and [Tableau Prep](https://www.tableau.com/support/releases/prep). Verify with Kyra or Nicole which versions to install. If you plan to publish a file to the CCAO's Tableau Server, the versions of Tableau Desktop and Tableau Prep [cannot be more recent](https://help.tableau.com/current/desktopdeploy/en-us/desktop_deploy_version_compat_top.htm) than the version of Tableau Server.
```

Comment:
**nitpick:** Let's not include names in the wiki, instead use the @ to tag them by username. Better yet, use @ to tag a [GitHub team](https://github.com/orgs/ccao-data/teams).